pages/test-evaluate/evaluation-tools/index.pl.md

```diff
@@ -0,0 +1,141 @@
+    <td>Osoba obok komputera ze stroną internetową używa lupy ze słowem „dostępność”.
+	</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia poprowadzą Cię przez te testy, których nie można zautomatyzować.</td>
+    <td>Lista kontrolna obok komputera.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia sprawdzają tylko jedną stronę na raz, inne mogą sprawdzać całe witryny.</td>
+    <td>Wyświetlane są znaczniki niepowodzenia, powodzenia i ostrzeżenia.
+	
+	</td>
+  </tr>
+  <tr>
+    <td>Narzędzia można integrować z różnymi środowiskami pracy. Na przykład z przeglądarką internetową, systemem zarządzania treścią (C-M-S) oraz w narzędziach programistycznych i wdrożeniowych.</td>
+    <td>Słowa „Przeglądarka internetowa”, „CMS” i „środowisko wdrożeniowe” wokół ikony narzędzi wyświetlanej na komputerze.</td>
+  </tr>
+  <tr>
+    <td>Wspierają różne role w zespole projektowym, takie jak autorów treści, twórców kodu, projektantów oraz właścicieli produktów.</td>
+    <td>Ikona narzędzi jest otoczona ikonami: długopisu, kodu, pędzla i osoby z kluczem.</td>
+  </tr>
+  <tr>
+    <td>Trzeba pamiętać, że w niektórych przypadkach narzędzia mogą dawać nieprawidłowe wyniki.</td>
+    <td>Lupa powiększa trójkątny znak z wykrzyknikiem. Wyświetlanych jest wiele lup. </td>
+  </tr>
+  <tr>
+    <td>Dlatego nie polegaj nadmiernie na tym, co mówią narzędzia w stosunku do tego, jakie są rzeczywiste doświadczenia osób korzystających z Twojej witryny.</td>
+    <td>Ekran dzieli się na 12 pól, w każdym inna osoba przed komputerem.</td>
+  </tr>
+  <tr>
+    <td>„Wybór narzędzi oceny dostępności cyfrowej” wyjaśnia, co narzędzia mogą, a czego nie mogą zrobić, oraz na co zwrócić uwagę, aby znaleźć narzędzia spełniające Twoje potrzeby.</td>
+    <td>Wybór narzędzi oceny dostępności cyfrowej. Pojawia się ikona skrzynki narzędziowe, następnie drogowskazu i lornetki.</td>
+  </tr>
+  <tr>
+    <td>„Lista narzędzi oceny dostępności cyfrowej” zawiera filtry, które pomogą Ci znaleźć odpowiednie narzędzie dla Twojej konkretnej sytuacji.</td>
+    <td>Wiele dokumentów ze słowem Narzędzia przewija się w dół do ikony filtra, z którego wychodzą tylko dwa dokumenty.</td>
+  </tr>
+  <tr>
+    <td>Z narzędziami i wiedzą, jak z nich prawidłowo korzystać, masz wszystko, czego potrzebujesz, aby skuteczniej znajdować problemy z dostępnością.</td>
+    <td>Osoba, przy niej ikony narzędzi i żarówki. Ikony łączą się ze sobą, tworząc lupę ze słowem „bariery”.</td>
+  </tr>
+  <tr>
+    <td>Dostępność internetowa: niezbędna dla niektórych, przydatna dla wszystkich.</td>
+    <td>Wokół komputera układają się ikony: ręki, oka, mózgu, ucha i ust z falą dźwiękową.</td>
+  </tr>
+  <tr>
+    <td>Aby uzyskać więcej informacji o narzędziach oceny dostępności, odwiedź stronę <a href="https://www.w3.org/WAI/evaluation">w3.org/WAI/evaluation</a>.</td>
```

Comment:
@zwiastunsw The original version does not contain a link. Please also keep the hyphens (w3.o-r-g/W-A-I/evaluation).